osutil/disks/disks_linux.go

```diff
@@ -649,10 +714,17 @@ func (d *disk) populatePartitions() error {
 				return fmt.Errorf("cannot parse device major number format: %v", err)
 			}
 
-			part.Minor, err = strconv.Atoi(udevProps["MINOR"])
+			maj, err := requiredUdevPropUint(udevProps, "MAJOR")
 			if err != nil {
-				return fmt.Errorf("cannot parse device major number format: %v", err)
+				return fmt.Errorf("cannot get required udev property for device %s (a partition of %s): %v", partDev, d.Dev(), err)
+			}
+			part.Major = int(maj)
+
+			min, err := requiredUdevPropUint(udevProps, "MINOR")
+			if err != nil {
+				return fmt.Errorf("cannot get required udev property for device %s (a partition of %s): %v", partDev, d.Dev(), err)
 			}
+			part.Minor = int(min)
```

Comment:
Nitpick, seems like Minor and Major should be made uints, or is it too annoying to change them?

Reply:
I added a TODO about this